requirements.txt

```diff
@@ -4,6 +4,7 @@ chardet
 future
 numpy
 pandas
+pycurl
```

Comment:
do we need this?

Reply:
Yeah, it became a requirement once I used this line (which was also required for the new proxy variables): 
```
AsyncHTTPClient.configure("tornado.curl_httpclient.CurlAsyncHTTPClient")
```

Reply:
otherwise you'll get the error: 
```
ModuleNotFoundError: No module named 'pycurl'
```